### PR TITLE
Started removing static members in math.ooc

### PR DESCRIPTION
**Pull request by marcusnaslund**
Will require changes in other files, so this is just for example. Is this what we want @simonmika ?

### CODE REVIEW

**Comment by simonmika** [comment]
It looks like the right way to go. Long term we will want unit tests as well.

**Comment by marcusnaslund** [comment]
I have already written a lot of unit tests for this, which have to be updated now also.

**Comment by marcusnaslund** [comment]
Care to take over peer review @sebastianbaginski ?

File: source/geometry/FloatPoint3D.ooc [hunk 1]
```diff
@@ -45,7 +45,7 @@ FloatPoint3D: cover {
 		This new(radius * (azimuth cos()) * (elevation sin()), radius * (azimuth sin()) * (elevation sin()), radius * (elevation cos()))
 	}
 	angles: static func (rx, ry, n: Float) -> This {
-		z := n*n sqrt() / (1 + ry tan() squared() + rx tan() squared())
+		z := n*n sqrt() / (1 + ry tan() squared + rx tan() squared)
```
Comment (sebastianbaginski) [inline source/geometry/FloatPoint3D.ooc]: Same here, `x^2` is a property, but `x^(1/2)` is not ?

File: source/math/FloatVectorList.ooc [hunk 2]
```diff
@@ -228,7 +228,7 @@ FloatVectorList: class extends VectorList<Float> {
 		result := This new(size)
 		factor := 1.0f / (sqrt(2.0f * Float pi) * sigma)
```
Comment (sebastianbaginski) [inline source/math/FloatVectorList.ooc]: `1.0f / (2.0f * Float pi) sqrt() * sigma` ?

File: source/math/FloatVectorList.ooc [hunk 3]
```diff
@@ -228,7 +228,7 @@ FloatVectorList: class extends VectorList<Float> {
...
 		for (i in 0 .. size)
-			result add((factor * pow(Float e, -0.5f * ((i - (size - 1.0f) / 2.0f) squared()) / (sigma squared()))) as Float)
+			result add((factor * pow(Float e, -0.5f * ((i - (size - 1.0f) / 2.0f) squared) / (sigma squared))) as Float)
```
Comment (sebastianbaginski) [inline source/math/FloatVectorList.ooc]: Why the `pow` global function call is left here ?

**Comment by sebastianbaginski** [comment]
There are some global function calls left. Do you want to take care of them here or another PR ? Otherwise looks good.

**Comment by marcusnaslund** [comment]
I don't want to make this PR too huge, already had one where I fixed some, I will make a third that takes care of the rest.

**Pull request merged**